build.gradle

```diff
@@ -562,6 +562,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind
```

Comment:
Cool, so for this JIRA shall I create a KIP for dragging the dependency into `kafka-clients`?